Create opcua-1021-9-0-opc-ua-device-gateway-with-thin-edge-mqtt-forwarding.md (#3574)
SoftwareAG/c8y-docs: content/change-logs/device-management/opcua-1021-9-0-opc-ua-device-gateway-with-thin-edge-mqtt-forwarding.md

---

> date: ""

Reviewer: ```suggestion
date: 2025-07-28
```

> title: OPC UA Device gateway with thin-edge Mqtt Forwarding

Reviewer 2: ```suggestion
title: OPC UA device gateway with Thin Edge MQTT forwarding
```

> title: OPC UA device gateway with Thin Edge MQTT forwarding

Reviewer 2: ```suggestion
title: OPC UA device gateway with Thin Edge MQTT Forwarding
```

product_area: Device management & connectivity
change_type:

>   - value: change-QHu1GdukP
>     label: Feature

Reviewer: ```suggestion
  - value: change-pXAlHAWka
    label: Preview
```

component:
  - value: component-Tf05_KQ-B
    label: OPC UA
build_artifact:
  - value: tc-MLn0oFRX-
    label: opcua
ticket: DM-4473
version: 1021.9.0
---

> Starting from opcua-device-gateway version 1021.9.0, the device gateway has ability to forward data from OPC-UA servers to thin-edge MQTT topics instead of sending them directly to Cumulocity. Primary use case is to use thin-edge also for data forwarding when running in thin-edge mode. While doing this, opcua-device-gateway bundles multiple collected measurement/event series into a single measurement/event when its from the same device protocol and using cyclic reads.

Reviewer 2: ```suggestion
Starting from the opcua-device-gateway version 1021.9.0, the OPC UA device gateway can forward data from OPC UA servers to Thin Edge MQTT topics instead of sending it directly to {{< product-c8y-iot >}}. The primary use case is to utilize Thin Edge also for data forwarding when running in Thin Edge mode. While doing this, the OPC UA device gateway bundles multiple collected measurement/event series into a single measurement/event when they are from the same device protocol and use cyclic reads.
```
